CHANGELOG/CHANGELOG-1.14.md

```diff
@@ -23,3 +24,4 @@ When cutting a new release, update the `unreleased` heading to the tag being gen
 * [CHANGE] Update Medusa to v0.20.1
 * [BUGFIX] [#1235](https://github.com/k8ssandra/k8ssandra-operator/issues/1235) Prevent operator from modifying the spec when superUserRef is not set. Also, remove the injection to mount secrets to cassandra container.
 * [BUGFIX] [#1239](https://github.com/k8ssandra/k8ssandra-operator/issues/1239) Use `concurrent_transfers` setting from MedusaConfiguration object if it is actually set.
+* [BUGFIX] [#1240](https://github.com/k8ssandra/k8ssandra-operator/issues/1240) The PullSecretRef for medusa is ignored in the standalone deployment of medusa
```

Comment:
**Issue:** 1.14.0 was released already and this line needs to go under a new `## unreleased` section to create above.

Reply:
Okay fixxed :)